clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -431,152 +433,151 @@ void foo1()
   //expected-warning@+2{{attribute 'bank_bits' is already applied}}
   [[intelfpga::bank_bits(42,43)]]
   [[intelfpga::bank_bits(1,2)]]
-  unsigned int bb_two[4];
+  unsigned int bb_bb[4];
 
   //expected-error@+1{{the number of bank_bits must be equal to ceil(log2(numbanks))}}
   [[intelfpga::numbanks(8), intelfpga::bank_bits(3,4)]]
-  unsigned int bb_three[4];
-
-  //expected-error@+1{{the number of bank_bits must be equal to ceil(log2(numbanks))}}
-  [[intelfpga::bank_bits(3,4), intelfpga::numbanks(8)]]
-  unsigned int bb_four[4];
+  unsigned int bb_numbanks[4];
 
   //expected-error@+1{{bank_bits must be consecutive}}
   [[intelfpga::bank_bits(3,3,4), intelfpga::bankwidth(4)]]
-  unsigned int bb_five[4];
+  unsigned int bb_noncons[4];
 
   //expected-error@+1{{bank_bits must be consecutive}}
   [[intelfpga::bank_bits(1,3,4), intelfpga::bankwidth(4)]]
-  unsigned int bb_six[4];
+  unsigned int bb_noncons_2[4];
```

Comment:
For the sake of consistency:
```suggestion
  unsigned int bb_noncons_1[4];
```

Reply:
let's count variable names from 1, not from 0? 😏